### DISCUSSION TITLE
Type hinting failed

### DISCUSSION THREAD

**Discussion by robov** [owen-it/laravel-auditing#782] [Q&A]
When I run this static analyser : https://github.com/imanghafoori1/laravel-microscope

I get: 
The type hint on the listener: OwenIt\Auditing\Listeners\RecordCustomAudit does not match the event class path

The RecordCustomAudits::handle accepts a `\OwenIt\Auditing\Contracts\Auditable`
but the event is not derived not implemeting this contract

**Comment by MortenDHansen** [comment]
Alright. Acknowledged. I do not know what it means though. Does it cause an issue i can proof?

**Comment by robov** [comment]
I have not used that custom section, so I cannot say if an how it can produce a bug.  But it might suddenly pop up somewhere

**Comment by parallels999** [comment]
hi @robov, feel free to make a PR for that, thanks
